ui/src/album/AlbumDetails.js

```diff
@@ -25,7 +25,7 @@ const useStyles = makeStyles((theme) => ({
   root: {
     [theme.breakpoints.down('xs')]: {
       padding: '0.7em',
-      minWidth: '24em',
+      minWidth: '20em',
```

Comment:
Is it still needed?

Reply:
Yeah, it is still required. Is there any issue with this, please let me know